docs/docs/clients/01-typescript.md

```diff
@@ -1,265 +1,365 @@
 ---
 sidebar_position: 1
-description: Information about the generated Typescript client code.
+description: Information about the generated TypeScript client code.
 ---
 
-# Typescript code generation
+# TypeScript frontend
 
-The `ignite generate ts-client` command generates a Typescript client for your blockchain project.
+Ignite offers powerful functionality for generating client-side code for your
+blockchain. Think of this as a one-click client SDK generation tailored
+specifically for your blockchain.
 
-Typescript client code can be automatically regenerated on reset or source code changes when the
-blockchain is started with the `--generate-clients` options.
+See [`ignite generate ts-client --help`](/cli#ignite-generate-ts-client) learn
+more on how to use TypeScript code generation.
 
-## Client code generation
+## Starting a node
 
-To configure TypeScript (TS) client code generation add the `client` settings to `config.yml`:
+For testing purposes add a new account to `config.yml` with a mnemonic:
 
-```yaml
-client:
-  typescript:
-    path: ts-client
+```yml title="config.yml"
+accounts:
+  - name: frank
+    coins: ["1000token", "100000000stake"]
+    mnemonic: play butter frown city voyage pupil rabbit wheat thrive mind skate turkey helmet thrive door either differ gate exhibit impose city swallow goat faint
 ```
 
-A TS client is generated in the `ts-client` directory when using this configuration.
+Run a command to generate TypeScript clients for both standard and custom Cosmos
+SDK modules:
 
-## Client code regeneration
+```
+ignite generate ts-client --clear-cache
+```
 
-To regenerate all clients for custom and standard Cosmos SDK modules, run this command:
+Run a command to start your blockchain node:
 
-```bash
-ignite generate ts-client
+```
+ignite chain serve -r
 ```
 
-## Setup
+## Setting up a TypeScript frontend client
 
-The best way to get started building with the TypeScript client is by using a [Vite](https://vitejs.dev/) boilerplate.
-Vite provides boilerplate code for vanilla TS projects as well as react, vue, lit, svelte and preact frameworks.
-You can find additional information at the [Vite Getting Started guide](https://vitejs.dev/guide/).
+The best way to get started building with the TypeScript client is by using a
+[Vite](https://vitejs.dev/) boilerplate. Vite provides boilerplate code for
+vanilla TS projects as well as React, Vue, Lit, Svelte and Preact frameworks.
+You can find additional information at the [Vite Getting Started
+guide](https://vitejs.dev/guide/).
 
-You will also need to polyfill the client's dependencies. The following is an example of setting up a vanilla TS
-project with the necessary polyfills.
+You will also need to polyfill the client's dependencies. The following is an
+example of setting up a vanilla TS project with the necessary polyfills.
 
 ```bash
 npm create vite@latest my-frontend-app -- --template vanilla-ts
+cd my-frontend-app
```

Comment:
since you use relative path to ts-client in the rest of the docs, maybe it's worth pointing out that `npm create` must be ran in your project root

Reply:
Not sure why you would need `npm create`. I've learned you need to install deps to make it work:

```
cd ts-client
npm install
```